static/js/components/EditProfileDisplay.js

```diff
@@ -83,28 +77,11 @@ export class EditProfilePage extends React.Component<Props> {
         <MetaTags>
           <title>{formatTitle(EDIT_PROFILE_PAGE_TITLE)}</title>
         </MetaTags>
-        <div className="row auth-header">
-          <h1 className="col-12">Edit Profile</h1>
-        </div>
-        <div className="auth-card card-shadow row">
-          <div className="container">
-            <div className="row">
-              <div className="col-12 auth-form">
-                {currentUser.is_authenticated ? (
-                  <EditProfileForm
-                    countries={countries}
-                    user={currentUser}
-                    onSubmit={this.onSubmit.bind(this)}
-                  />
-                ) : (
-                  <div className="row">
-                    You must be logged in to edit your profile.
-                  </div>
-                )}
-              </div>
-            </div>
-          </div>
-        </div>
+        <EditProfileForm
```

Comment:
Another nitpick which may not be relevant but city and state are swapped in the design compared to here